sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowJsonCoderTest.java

```diff
@@ -67,6 +68,13 @@ public void testDecodeEncodeEqual() throws Exception {
     }
   }
 
+  @Test
+  public void testLargeRow() throws Exception {
```

Comment:
Before: 

```
com.fasterxml.jackson.core.exc.StreamConstraintsException: String length (20054016) exceeds the maximum length (20000000)
	at com.fasterxml.jackson.core.StreamReadConstraints.validateStringLength(StreamReadConstraints.java:324)
	at com.fasterxml.jackson.core.util.ReadConstrainedTextBuffer.validateStringLength(ReadConstrainedTextBuffer.java:27)
	...
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3740)
	at org.apache.beam.sdk.io.gcp.bigquery.TableRowJsonCoder.decode(TableRowJsonCoder.java:59)
	...
	at org.apache.beam.sdk.testing.CoderProperties.coderDecodeEncodeEqual(CoderProperties.java:97)
	at org.apache.beam.sdk.io.gcp.bigquery.TableRowJsonCoderTest.testLargeRow(TableRowJsonCoderTest.java:75)
```

after:

testLargeRow	0.615s	passed

Also tested with example Pipeline (read TableRow from BigQuery with large row, then ReShuffle it).